integration-tests/spring-extension-support-test/src/main/java/com/example/application/autogrid/UserRepository.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2023 Marco Collovati, Dario Götze
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.application.autogrid;
+
+import org.springframework.data.repository.CrudRepository;
```

Comment:
spring data?

Reply:
Yes, exactly. It is required for Hilla AutoGrid support